notebooks/Datasets_3_IC50_conversion.ipynb

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
<p>Excellent idea, great for traceability.&nbsp; One thing I would be interested in, even if it's just output in the notebook, is some sort of idea how many of the assays actually have a DOI, and how many don't.&nbsp; I can't tell from this code or the csv.  One way would be to create a count and print the % of assays with a DOI, the other way would be to have every assay in the output csv, and if it doesn't have a DOI, it gets an empty string, or "not found" or something.</p><p> </p><p>I prefer the second approach because it is explicit and so would make traceability easier.&nbsp;If we're tracking something down in the future, and we see there's no entry for that assay id, that's more confusing than if the assay id is there and says "not found."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
ok, I don't think this is high priority and we don't necessarily need this in master, so keeping it as a branch for now